public_suffix_list.dat

```diff
@@ -11216,6 +11216,10 @@ ltd.ua
 // 611coin : https://611project.org/
 611.to
 
+// AAA workspace : https://aaa.vodka
+// Submitted by Kirill Rezraf <admin@aaa.vodka>
```

Comment:
Hello @rezraf just a heads-up that the email address you left in the comment section is not valid, as the domain of your email address doesn't have MX records, so it won't be reachable if the PSL maintainers or volunteers try to contact you or your organization in the future. It's recommended to either fix the email account or submit an amendment to update it to a valid one. Thank you.